newm-server/src/main/kotlin/io/projectnewm/server/ext/LocalDateTimeExt.kt

```diff
@@ -0,0 +1,7 @@
+package io.projectnewm.server.ext
+
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.util.Date
+
+fun LocalDateTime.toDate(): Date = Date.from(atZone(ZoneId.systemDefault()).toInstant())
```

Comment:
Do you think we should make this date UTC?

Reply:
There is another way to do this conversion, which is more intuitive to the fact that is UTC. I'll change to that.